src/style.css

```diff
@@ -0,0 +1,89 @@
+* {
```

Comment:
Went beyond the project requirements 🤯 Nice job!

Reply:
Thank you!